src/MSAL.PCL/Internal/Http/HttpClientFactory.cs

```diff
@@ -35,7 +35,17 @@ internal class HttpClientFactory
         private static HttpClient _client;
         private static readonly object LockObj = new object();
         public static bool ReturnHttpClientForMocks { set; get; }
+        public const long MaxResponseContentBufferSizeInBytes = 1024*1024;
 
+        private static HttpClient CreateHttpClient()
+        {
+            var httpClient = new HttpClient(HttpMessageHandlerFactory.GetMessageHandler(ReturnHttpClientForMocks))
+            {
+                MaxResponseContentBufferSize = MaxResponseContentBufferSizeInBytes
+            };
+
+            return httpClient;
+        }
```

Comment:
insert new line after }.
It is a good practice to run resharper code reformat to align curly braces as well.